common/scaffolding/reports2/text-level-en.html.html

```diff
@@ -0,0 +1,151 @@
+
+<html>
+<body>
+<nav>
+<b> (a element)</b>
+ <ul>
+  <li> <a href="/">aliquid</a> </li>
+  <li> <a href="/news">News</a> </li>
+  <li> <a>Examples</a> </li>
+  <li> <a href="/legal">Legal</a> </li>
+ </ul>
```

Comment:
I don't think we need an unordered list item here, simple a series of paragraph will do the work.

And let's have a few common example variation which I added in this code suggestion

```suggestion
  <p><code>href</code> links: <a href="/">aliquid</a> </p>
  <p id="anAnchor"><code>href</code> anchor: <a href="#anAnchor">News</a> </p>
  <p><code>href</code> empty: <a href>aliquid</a> </p>
  <p>No attribute: <a>Examples</a> </p>
  <p><code>hreflang</code> set to French <a href="/legal" hreflang="fr">Legal</a> </p>
  <p><code>hreflang</code> set to English <a href="/legal" hreflang="en">Lorem</a> </p>
  <p><code>lang</code> set to French <a href="/legal" lang="fr">Ipsum</a> </p>
  <p><code>lang</code> set to English <a href="/legal" lang="en">dolor</a> </p>
```